examples/set-namespace/advanced/README.md

```diff
@@ -41,7 +41,7 @@ configuration.
 
 Suppose we have a CRD with group `dev.example.com`, version `v1` and kind
 `MyResource`. We want the namespace to be set in field
-`.spec.configmapRef.namespace` as well. We specify it in field `fieldSpecs`.
+`.spec.configmapRef.namespace` as well. We specify it in field `additionalNamespaceFields`.
 
```

Comment:
We can introduce `additionalNamespaceFields` with:

`If you have a resources that uses `namespace` in fields other than `metadata.namespace`, you can use `additionalNamespaceFields` to update those namespace fields. WDYT ?